Components/Grids/grid.css

```diff
@@ -0,0 +1,29 @@
+.grid-col-2{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    width: 70%;
+    background-color: var(grey-clr);
+    min-width: 300px;
+}
+.grid-col-3{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    width: 70%;
+    background-color: var(grey-clr);
+    min-width: 300px;
```

Comment:
Convert px to rem. Since you are already using rem at different places. Try to use same units everywhere. For values less than 15 px, you can use px directly.